tools/compare-values-schema-content.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""
+This script is meant to assist in a manual validation that the content of
+schema.yaml covers values.yaml, and vice versa.
+
+FIXME: It would be nice to run this as part of our CI pipeline to report if
+       schema.yaml and values.yaml gets out of sync, but first we need to
+       address what it means to be out of sync.
+
+       Consider if schema.yaml describes extraLabels, and we in this helm chart
+       have an extra label set in values, how should our comparison realize that
+       its nothing to bother about?
+
+       That kind of complexity is currently an issue for labels, resources,
+       containerSecurityContext, readiness- and livenessProbe's, and hub.config.
+"""
+
+import jsonschema
+import os
+import sys
+
+from collections.abc import MutableMapping
+
+import yaml
+
+# Change current directory to this directory
+os.chdir(os.path.dirname(sys.argv[0]))
```

Comment:
Also see comment below in validate-against-schema  - I don't think `os.chdir` in scripts is usually something we want to do.

Reply:
I'd like the script to function the same way independently of where they are executed from.

I guess the crux of using os.chdir is that it can influence the environment calling the script? Running it from my terminal didn't influence the terminal, but it could perhaps?

I'm not sure on the path forward here, suggestions?

Reply:
Resolved by cb0ae7d9